frontend/src/components/ui/question/QuestionBox.js

```diff
@@ -11,24 +11,38 @@ import QuestionCategories from "./QuestionCategories";
 import QuestionTitle from "./QuestionTitle";
 import Question from "./Question";
 
-import styles from './QuestionBox.module.css'
+import styles from "./QuestionBox.module.css";
 
 /* 
   Find out which difficulty user chose,
   obtain a Question of that difficulty from
   Question service and display it
 */
 
-function QuestionBox() {
+function QuestionBox(props) {
   const [title, setTitle] = useState("");
   const [categories, setCategories] = useState([]);
   const [content, setContent] = useState("");
   const [difficulty, setDifficulty] = useState("");
+  const [roomId, setRoomId] = useState("");
+  const [questionNumber, setQuestionNumber] = useState();
 
   useEffect(() => {
     async function fetchData() {
```

Comment:
Good should for abstracting out async function and call it in useEffect instead of making useEffect's anonymous function asynchronous. It's a good practice 😸

Reply:
Thanks! :)